src/index.css

```diff
@@ -1,47 +1,151 @@
-.header {
+@import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Oswald:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Abel&display=swap');
+
+body {
```

Comment:
- [ ] Well done so far styling the app. However, and as the previous reviewer required, please follow the given design in the following aspects
- Colors
- Spacings (margins and padding)
- Font sizes
- Font styles
- Font family
Keep up, you're almost there 💪 🥇

Reply:
changed my font to` Montserrat` and `Roboto`